1.	The time required for servicing transmissions is normally distributed with mu = 45 minutes and std= 8 minutes. The service manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager cannot meet his commitment? 

the total time requires fro servicing is 1 hour,
the service manager takes a car for work 10 minutes after 
time left to work is (60-10)= 50 minutes

P(X>50)=1-P(x<=50)

In [1]:
from scipy import stats

In [2]:
#Probability of a service manager cannot meet his commitment
a=1-stats.norm.cdf(50,45,8) 
print (a)

0.26598552904870054


 ### A.	More employees at the processing center are older than 44 than between 38 and 44.


older than 44 = P(X>44)
between 38 and 44 = P(38=<x<=44)= P(x>=38)-P(x<=44)

In [3]:
a = stats.norm.cdf(44,38,6)
print (a)
b= 1-a 
## peoples are older than 44
print(b)
## peoples are between 38 and 44 
## 38 is the mean of distribution its half of the distribution so , it takes 0.5
middle = a-0.5  
print(middle)

0.8413447460685429
0.15865525393145707
0.3413447460685429


##### the statement  A is false because of more people in between 38 ad 44(34.13%) and less people are older than 44 (15.86%)

### B.	A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.

In [4]:
## zscore for 30
stats.norm.cdf(30,38,6)


0.09121121972586788

In [5]:
## zscore * population = number of employees 
stats.norm.cdf(30,38,6)*400


36.484487890347154

##### statement B is true  under the age of 30 employees are attract to the training centre

#### 4.	Let X ~ N(100, 20^2). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99. 



In [6]:
stats.norm.ppf(0.005)

-2.575829303548901

In [7]:
stats.norm.ppf(.995)

2.5758293035489004

In [8]:
## (mu - zscore* std deviation ) = value of x ,lower limit
100+(-2.57*20)

48.6

In [9]:
## (mu + Z score* std deviation) = value of x, upper limit
100+(2.57*20)

151.4

#### A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company
#### total profit of company = (5, 3^2) +(7, 4^2) = (12, 5^2)

In [10]:
## finding the lower limit zscore
stats.norm.ppf(0.025)

-1.9599639845400545

In [11]:
## finding the upper limit of zscore
stats.norm.ppf(0.975)

1.959963984540054

In [12]:
## upper and lower limit value of 95% range in $ million
(12+(1.96*5) , 12+(-1.96*5)) 

(21.8, 2.1999999999999993)

In [13]:
## upper and lower limit value of 95% range in Million of Rs.
Rs=("Rs.",2.2*45,"M","Rs.",21.8*45,"M")
print(Rs)

('Rs.', 99.00000000000001, 'M', 'Rs.', 981.0, 'M')


#### B.	Specify the 5th percentile of profit (in Rupees) for the company
 

In [14]:
## zscore value of the 5th percentile
stats.norm.ppf(0.05)

-1.6448536269514729

In [15]:
  ## to find the profit of 5th percentile we use the zscore formula
 ## (p-12)/5 = -1.644
p =(12-(5*1.644))
print ("Rs",p*45,"M")
## 5th percentile profit = Rs 170 M

Rs 170.10000000000005 M


#### c.	Which of the two divisions has a larger probability of making a loss in a given year?


In [16]:
## p<0

In [17]:

stats.norm.ppf(0.025)

-1.9599639845400545

In [18]:
stats.norm.ppf(0.975)

1.959963984540054

In [19]:
## the first division tested on left area 0.025
stats.norm.cdf(-1.96,5,3)*100 

1.0170438668719681

In [20]:
## the second division tested  on left area 0.025
stats.norm.cdf(-1.96,7,4)*100

1.2545461435946563

from the above conclusion of first and second division left area we state that,

The first division of company, thus have larger probability of making a loss in a given year.

